.env

```diff
@@ -11,3 +11,11 @@ MYSQL_PORT="3306"
 MYSQL_USER="root"
 MYSQL_PASSWORD="secret"
 MYSQLDUMP_DATABASE="my_wiki"
+CRON_TIME_HOURLY="0 */1 * * *"
```

Comment:
@vijayabharathib actually we don't need to specify here. the default value will loaded from Dockerfile. 

Moreover, why we need to allow changing the cron for hourly

Reply:
Ok fine.